horovod/spark/torch/remote.py

```diff
@@ -139,6 +139,12 @@ def train(serialized_model, optimizer_cls, model_opt_state_serialized,
             print("Skip pinning current process to the GPU.")
 
         cuda_available = torch.cuda.is_available() and should_pin_gpu
```

Comment:
The last bit of the condition has to go:
```suggestion
        cuda_available = torch.cuda.is_available()
```